deluge/core/torrentmanager.py

```diff
@@ -279,19 +279,14 @@ def update(self):
                 'Paused',
                 'Queued',
             ):
-                # If the global setting is set, but the per-torrent isn't...
-                # Just skip to the next torrent.
-                # This is so that a user can turn-off the stop at ratio option on a per-torrent basis
-                if not torrent.options['stop_at_ratio']:
```

Comment:
Why was this removed?

Reply:
It was never activated, because we are inside an if block that already checks it:
https://github.com/deluge-torrent/deluge/blob/b59549eaf2a222679a7cc53d8a95dbb63c0daf36/deluge/core/torrentmanager.py#L276